src/instructlab/configuration.py

```diff
@@ -1124,9 +1124,10 @@ def map_train_to_library(ctx, params):
     )
 
     lora_args = LoraOptions()
+    lora = False
     if params["lora_rank"] is not None:
-        lora = True
-        lora_args.rank = params["lora_rank"]
+        if not lora:
```

Comment:
You can just keep `lora = True` here because it doesn't really benefit us to be performing the `if not lora` check prior to setting it. 

So all of the blocks should now be set to:

```diff
- if not lora:
-   lora = True
+ lora = True
```

Reply:
Updated all the blocks!